Bender.yml

```diff
@@ -85,26 +88,48 @@ sources:
   # levels 1 and 0, etc. Files within a level are ordered alphabetically.
   # Level 0
   - hw/carfield_pkg.sv
+  - hw/carfield_chip_pkg.sv
   - hw/regs/carfield_reg_pkg.sv
   - hw/regs/carfield_reg_top.sv
+#  - target/sim/src/behav-padframe/src/pkg_carfield_padframe_behav.sv
+#  - target/sim/src/behav-padframe/src/pkg_internal_carfield_padframe_behav_periph.sv
+#  - target/sim/src/behav-padframe/src/carfield_padframe_behav_periph_config_reg_pkg.sv
+#  - target/sim/src/behav-padframe/src/carfield_padframe_behav_periph_config_reg_top.sv
+#  - target/sim/src/behav-padframe/src/carfield_padframe_behav_periph_pads.sv
+#  - target/sim/src/behav-padframe/src/carfield_padframe_behav_periph_muxer.sv
+#  - target/sim/src/behav-padframe/src/carfield_padframe_behav_periph.sv
+#  - target/sim/src/behav-padframe/src/carfield_padframe_behav.sv
+  - hw/padframe/src/pkg_astral_padframe.sv
+  - hw/padframe/src/pkg_internal_astral_padframe_periph.sv
+  - hw/padframe/src/astral_padframe_periph_config_reg_pkg.sv
+  - hw/padframe/src/astral_padframe_periph_config_reg_top.sv
+  - hw/padframe/src/astral_padframe_periph_pads.sv
+  - hw/padframe/src/astral_padframe_periph_muxer.sv
+  - hw/padframe/src/astral_padframe_periph.sv
+  - hw/padframe/src/astral_padframe.sv
+  - hw/padframe/pad_behav.sv
   # Level 1
   - hw/cheshire_wrap.sv
   - hw/hyperbus_wrap.sv
   - hw/l2_wrap.sv
   - hw/carfield_rstgen.sv
   # Level 2
   - hw/carfield.sv
+  - target/synth/src/carfield_synth_wrap.sv
 
   - target: spatz
     files:
       - hw/spatz_cluster_wrapper.sv
 
   - target: test
     files:
+      #- target/sim/src/behav-padframe/pad_behav.sv
       - target/sim/src/hyp_vip/s27ks0641.v
       - target/sim/src/vip_carfield_soc.sv
       - target/sim/src/carfield_fix.sv
       - target/sim/src/carfield_tb.sv
+      - target/sim/src/carfield_synth_fix.sv
+      - target/sim/src/carfield_synth_tb.sv
```

Comment:
Would be good if we could directly replace the carfield_fix and carfield_tb with the new ones instead of adding files on top.

Reply:
Agree. When I finish my work I will update the file structure, the same way I did for the padframe that is now in the "hw" folder as opposed to "target/sim/src..."

Reply:
Done. Check the latest commits.